1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.


Считаем по формуле из методички (отклонение = 16 и sqrt(256) сокращаются). Если я верно понял "с", выходит вот так: 
Т1,2 = 80+-0.95

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [13]:
import numpy as np
import math

X = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
M = np.mean(X)
xs = np.std(X)
print('Среднее - ', M, 'Отклонение - ', xs)

Среднее -  6.590000000000001 Отклонение -  0.42766809560686203


T1,2 = 6.59 +- 0.42/sqrt(10) = 6.59 +- 0.13

3,4 задачи решать через тестирование гипотезы

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

z(табличное) = 1.645
z(рассчетное) = 0.5/(sigma/sqrt(n)

In [15]:
z = 0.5/(2/math.sqrt(10))
z

0.7905694150420948

In [ ]:
Оставляем нулевую гипотезу

4.Продавец утверждает, что средний вес пачки печенья составляет 200 г. 
Из партии извлечена выборка из 10 пачек. 
Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. 
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

**Дважды смотрел вебинар, не сообразил как решить эту задачу**